src/components/MermaidDash/components/LoadingBar.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import theme from '../../../styles/theme'
+
+const LoadingBarContainer = styled.div`
+  position: absolute;
+  left: 0px;
+  bottom: 0px;
+  background-color: ${theme.color.callout};
+  height: 4px;
+  width: ${({ width }) => width}%;
+  opacity: ${({ showLoadingBar }) => (showLoadingBar ? 1 : 0)};
+  transition:
```

Comment:
This transition does nothing because there's no duration associated with it. Try `1s`. 
Also the `opacity` property seems to do nothing here either.  

https://developer.mozilla.org/en-US/docs/Web/CSS/transition

Reply:
you are right, I add back 1s, it looks nicer with transition